### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by kachkaev**
* Bump [chalk](https://www.npmjs.com/package/chalk) (`^2.4.1` → `^2.4.2`)

* Bump [dotenv](https://www.npmjs.com/package/dotenv) (`^6.2.0` → `^8.2.0`) 
  [CHANGELOG.md](https://github.com/motdotla/dotenv/blob/master/CHANGELOG.md)

* Deduplicate dependencies in `yarn.lock` via [yarn-deduplicate](https://www.npmjs.com/package/yarn-deduplicate)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,8 +45,8 @@
   "author": "Aaron Franks",
   "license": "MIT",
   "dependencies": {
-    "chalk": "^2.4.1",
-    "dotenv": "^6.2.0",
+    "chalk": "^2.4.2",
+    "dotenv": "^8.2.0",
```
Comment (SimenB) [inline package.json]: this has dropped node v6, so when we merge this it should be released in a major

Comment (kachkaev) [reply package.json]: Node 8.12 is already required since https://github.com/SimenB/envalid/pull/12

**Comment by af** [comment]
👍

**Comment by kachkaev** [comment]
@af WDYT of making a release that drops Node 6 support?

**Comment by af** [comment]
@kachkaev Yeah that's the plan! Haven't been able to get to it the last few days but I'll try to soon, sorry for the delay

**Comment by af** [comment]
v6.0.0 is now out! Let me know if it's anything but smooth sailing ⛵️

**Pull request merged**